lib/tasks/evm_automate.rake

```diff
@@ -148,14 +148,26 @@ namespace :evm do
         elsif ENV['YAML_FILE'].present?
           puts "Importing automate domain: #{ENV['DOMAIN']} from file #{ENV['YAML_FILE']}"
           import_options['yaml_file']   = ENV['YAML_FILE']
+        elsif ENV['GIT_URL'].present?
+          puts "Importing automate domain from url #{ENV['GIT_URL']}"
+          import_options['url'] = ENV['GIT_URL']
+          import_options['overwrite'] = true
+          import_options['userid'] = ENV['USERID']
+          import_options['password'] = ENV['PASSWORD']
+          import_options['ref'] = ENV['REF'] || MiqAeDomain::DEFAULT_BRANCH
+          import_options['ref_type'] = ENV['REF_TYPE'] || MiqAeDomain::BRANCH
```

Comment:
This can be a literal hash which would read better:

```ruby
import_options = {
  'url'       => ENV['GIT_URL']
  'overwrite' => true
  'userid'    => ENV['USERID']
  'password'  => ENV['PASSWORD']
  'ref'       => ENV['REF'] || MiqAeDomain::DEFAULT_BRANCH
  'ref_type'  => ENV['REF_TYPE'] || MiqAeDomain::BRANCH
}
```